manifests/fpm/config.pp

```diff
@@ -104,14 +105,16 @@
     mode    => '0644',
   }
 
-  ensure_resource('file', '/var/run/php-fpm',
-    {
-      ensure => directory,
-      owner  => 'root',
-      group  => $root_group,
-      mode   => '0755',
-    }
-  )
+  if $manage_run_dir {
+    ensure_resource('file', '/var/run/php-fpm',
+      {
+        ensure => directory,
+        owner  => 'root',
+        group  => $root_group,
+        mode   => '0755',
+      }
+    )
```

Comment:
If we can skip resource management, I guess we can prefer using actual resources rather that relying on `ensure_resource()`:

```suggestion
    file { '/var/run/php-fpm':
      ensure => directory,
      owner  => 'root',
      group  => $root_group,
      mode   => '0755',
    }
```